_posts/0000-01-04-dileeppandiya.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hi How are you .?
```

Comment:
Your caption could describe the image shown here:
https://github.com/githubschool/way-to-go/pull/17/files#diff-008fd807f45701a5595c663d663380ecR7-R9